deploy/chart/templates/06-installplan.crd.yaml

```diff
@@ -41,3 +41,18 @@ spec:
               - Automatic
               - Manual
               - Update-Only # Will only apply an update if it updates existing packages only and doesn't add any new ones
+            approved:
+              type: boolean
+      anyOf:
```

Comment:
Found this here: https://stackoverflow.com/questions/38717933/jsonschema-attribute-conditionally-required/38781027

Reply:
We use that for the install strategy on CSV as well 👍